### PR TITLE
Video support

### PR DESCRIPTION
**Pull request by ozgrozer**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.0.22",
+  "version": "1.0.23",
```
Comment (ccoVeille) [inline package.json:2]: Bringing such feature should be a minor, not a patch of me ou follow semver.

So IG should have been a

```suggestion
  "version": "1.1.0",
```

**Review by ccoVeille — COMMENTED** [review]
Amazing feature !!! Thanks

File: package.json (hunk 1 above)
Comment (ozgrozer) [reply package.json:2]: Yeah but I just didn't follow semver from the beginning. I've added the image support later and it should've been v1.1.0 and now with the video should've been v1.2.0. But it doesn't really important I guess. I'm just trying out things. Not sure this video feature is going to work for everyone.

Comment (ccoVeille) [reply package.json:2]: OK, thanks for replying me

**Pull request merged**